### PR TITLE
Ignore missing imports for parameterized.

### PR DESCRIPTION
**Pull request by clokep**
This seems to be from a conflict between #11282 and #11228.

Example failing run: https://github.com/matrix-org/synapse/runs/4154822727?check_suite_focus=true

### CODE REVIEW

**Review by DMRobertson — APPROVED** [review]
LGTM assuming CI passes.

Sadly there aren't any type stubs in typeshed for parameterized.

I wonder why this didn't show up in #11282?

**Comment by clokep** [comment]
> I wonder why this didn't show up in #11282

It was based on #11263, so before #11228 merged.

(Visible via `refs/pull/11282/head`.)

**Pull request merged**
